packages/browserslist-config/README.md

```diff
@@ -20,6 +20,12 @@ Add this to your `package.json` file:
 ]
 ```
 
+Alternatively, add this to `.browserslistrc` file:
```

Comment:
They mention also `browserslist` file in docs https://github.com/browserslist/browserslist#config-file:

> Browserslist config should be named `.browserslistrc` or `browserslist` and have browsers queries split by a new line. Comments starts with # symbol:...